docs/cli-arguments.md

```diff
@@ -32,7 +32,7 @@ Usage of ./kube-state-metrics:
   -h, --help                              Print Help text
       --host string                       Host to expose metrics on. (default "::")
       --kubeconfig string                 Absolute path to the kubeconfig file
-      --labels-metric-allow-list string   Allows to pass a list of additional Kubernetes label keys that will be used in the resource' labels metric. By default the metric contains only name and namespace labels. To include additional labels provide a list of resource names in their plural form and Kubernetes label keys you would like to allow for them (Example: '=namespaces=["k8s-label-1","k8s-label-n",...],pods=["app"],...)'
+      --labels-metric-allow-list string   Allows to pass a list of additional Kubernetes label keys that will be used in the resource labels metric. By default the metric contains only name and namespace labels. To include additional labels provide a list of resource names in their plural form and Kubernetes label keys you would like to allow for them (Example: '=namespaces=[k8s-label-1,k8s-label-n,...],pods=[app],...')
```

Comment:
Thanks for the PR! You need to modify this in the https://github.com/kubernetes/kube-state-metrics/blob/b4fb3d474bb4d1562bb11a62ae114b4c4291f582/pkg/options/options.go#L102 and afterwards run the `make generate` target, that will regenerate this file.

Do you mind also to aim this PR against release-2.0 branch, and then we can port the changes to master. As the issue originates there. Thank you! 🎉

Reply:
Thank you for explaining how to do it in an easy-to-understand